quarkus/runtime/src/main/java/org/keycloak/quarkus/runtime/configuration/PropertyMappingInterceptor.java

```diff
@@ -102,7 +104,17 @@ public Iterator<String> iterateNames(ConfigSourceInterceptorContext context) {
             }
             allMappers.remove(mapper);
 
-            if (!mapper.hasWildcard()) {
+            if (mapper.hasWildcard()) {
```

Comment:
It should be possible to encapsulate this logic if instead of mapper.getTo() we used something like mapper.getConnectedTo()

Reply:
@shawkins Thanks for the review! 

If you have in mind a better approach, feel free to send a PR to my fork, or even create a new one with better changes - it's welcomed :)

Reply:
Improved handling of the connected wildcard mappers: https://github.com/keycloak/keycloak/pull/41026/commits/56c2c247365f933cda158249027ea63c2b7f6aa5